metasploit-framework.gemspec

```diff
@@ -44,6 +44,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^spec/})
   spec.require_paths = ["lib"]
 
+  spec.required_ruby_version = '>= 1.9'
```

Comment:
```suggestion
  spec.required_ruby_version = '>= 2.5'
```
Our minimum supported version is 2.5, though many things may work in lower versions we use a few methods from 2.5 and require some of the SSL and Integer fixes from the last couple years.

Reply:
That's totally fair.

Reply:
Great thank you @acammack-r7. Updated.